plugins/kubernetes-backend/src/service/KubernetesClientProvider.ts

```diff
@@ -52,6 +64,20 @@ export class KubernetesClientProvider {
       contexts: [context],
       currentContext: context.name,
     });
+
+    return kc;
+  }
+
+  // visible for testing
+  getKubeConfig(clusterDetails: ClusterDetails) {
+    let kc;
+
+    if (clusterDetails.kubeConfigContext) {
```

Comment:
shouldn't this be `clusterDetails.kubeConfigContext || clusterDetails.kubecConfigFile` I guess you can have default context in the file right?